scripts/rr_completion.zsh

```diff
@@ -0,0 +1,37 @@
+_rr() {
+  # allow overridding rr with another command (e.g. if multiple versions are installed)
+  zstyle -s :completion${curcontext}options command rr
+  : ${rr:=rr}
+
+  _rr_subcommands() {
+    $rr --list-commands | cut -s -d ' ' -f 3
+  }
+  _rr_traces() {
+    $rr ls | grep -v '^cpu_lock$'
```

Comment:
this `grep -v` isn't strictly necessary now that rr doesn't show cpu_lock in `rr ls`. but in older versions, cpu_lock was the only file shown that wouldn't be a valid trace. so it would be nice to keep this so that the script is compatible with older versions too.

Reply:
I don't see the value in having the script be compatible with older versions but I guess this is harmless.